firedrake/output/paraview_reordering.py

```diff
@@ -2,6 +2,14 @@
 import numpy as np
 from pyop2.utils import as_tuple
 
+try:
+    import vtkmodules.vtkCommonDataModel
+    del vtkmodules.vtkCommonDataModel
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError(
+        "Error importing vtkmodules, you may need to install by executing\n\t"
```

Comment:
I feel like we could give the user a slightly better message. Something like:

"Cannot import vtkmodules. Firedrake does not install VTK by default so you may need to install it by running pip install vtk first."

I feel that the current message doesn't sound deliberate enough.